playbooks/group_vars/web.yml

```diff
@@ -0,0 +1,78 @@
+---
+xnat:
+  owner: "{{ tomcat_owner }}"
+  group: "{{ tomcat_group }}"
+  install_downloads: "{{ tomcat_catalina_home }}/install_downloads"
+  pipeline_install_dir: "{{ tomcat_catalina_home }}/pipeline_installer"
+  processingUrl: ""
+
+xnat_source:
+  war_file_name: "xnat-web-{{ xnat_version }}.war"
+  plugins_downloads_dir: "/ansible/downloads/xnat_plugins"
+  xnat_downloads_dir: "/ansible/downloads/xnat"
+  pipeline_installer_file_name: "pipeline-installer-{{ xnat_pipeline_version }}.tar"
+  xnat_war_url: "https://api.bitbucket.org/2.0/repositories/xnatdev/xnat-web/downloads/xnat-web-{{ xnat_version }}.war"
+  pipelines_url: "https://api.github.com/repos/NrgXnat/xnat-pipeline-engine/tarball/{{ xnat_pipeline_version }}"
+  context_file_location: "/usr/share/tomcat/webapps/ROOT/META-INF/context.xml"
+
+# mirsg.infrastructure.tomcat
+tomcat_version: 9.0.82
+tomcat_owner: "tomcat"
+tomcat_group: "tomcat"
+
+tomcat_webapp_name: ROOT
+tomcat_root: "/usr/share/tomcat/webapps/{{ tomcat_webapp_name }}"
+tomcat_root_webapp: "{{ tomcat_root }}.war"
+
+tomcat_catalina_home: /usr/share/tomcat
+tomcat_catalina_opts: >-
+  -Dxnat.home={{ xnat_home_dir }}
+  -Xms{{ java_mem.Xms | default("512M") }}
+  -Xmx{{ java_mem.Xmx | default("1G") }}
+  -XX:MetaspaceSize={{ java_mem.MetaspaceSize | default("100M") }}
+  -XX:+UseG1GC
+  -server
+
+tomcat_hostname: localhost
+tomcat_port: 8080
+
+tomcat_backup_directory: /usr/share/tomcat_bkp
+tomcat_items_to_restore:
+  - "{{ tomcat_backup_directory }}/webapps"
+  - "{{ tomcat_backup_directory }}/logs"
+  - "{{ tomcat_backup_directory }}/install_downloads"
+  - "{{ tomcat_backup_directory }}/.postgresql"
+
+postgresql_client:
+  server_certificate_filename: "/usr/share/tomcat/.postgresql/root.crt" # where to copy the server certificate to on the client
+
+# mirsg.infrastructure.ssl_certificates
+postgresql_client_ssl_certificate:
+  owner: "{{ tomcat_owner }}"
+  group: "{{ tomcat_owner }}"
+  certificate_directory: "/usr/share/tomcat/.postgresql"
+  privatekey_filename: "/usr/share/tomcat/.postgresql/postgresql.key"
+  use_pk8: true
+  pk8_filename: "/usr/share/tomcat/.postgresql/postgresql.pk8"
+  csr_filename: "/usr/share/tomcat/.postgresql/postgresql.csr"
+  csr_common_name: "{{ xnat_web_server.host }}"
+  certificate_filename: "/usr/share/tomcat/.postgresql/postgresql.crt"
+  provider: "selfsigned"
+  cache_filename: "{{ database_client_certificate_cache_filename }}" # where to store the client certificate in cache
+
+java:
+  keystore_path: "/usr/lib/jvm/jre/lib/security/cacerts/"
+
+ldap_ca_cert_file_on_client: "{{ xnat.install_downloads }}/certs/ldap-ca.cert"
+
+# mirsg.infrastructure.firewalld
+firewalld_internal_zone_open_services:
+  - http
+  - https
+  - ssh
+firewalld_public_zone_open_services:
+  - http
+  - https
+firewalld_work_zone_open_services:
+  - http
+  - https
```

Comment:
do these need to be updated in the [xnat collection](https://github.com/UCL-MIRSG/ansible-collection-xnat/blob/38d48cea16c768d73f803de2019c0c80cefe733b/playbooks/group_vars/web.yml#L68-L78) for it to work correctly?

Reply:
They do - can't believe I missed those. I've opened [another PR](https://github.com/UCL-MIRSG/ansible-collection-xnat/pull/20).

Reply:
thanks!